### PR TITLE
CHI-1406: Fix accessibility for CSAM dropdown & add clickaway dismiss

### PR DESCRIPTION
**Pull request by stephenhand**
## Description

* Use HTML button tag for CSAM type dropdown for correct accessibility behaviour
* Add clickaway dismissal to CSAM type dropdown

### Checklist
- [X] Corresponding issue has been opened
- N/A New tests added
- [X] Feature flags added
- [X] Strings are localized
- [X] Tested for chat contacts
- [X] Tested for call contacts

### Related Issues
Fixes CHI-1406

### Verification steps

* See ticket comments
* Also verify keyboard interactions with CSAM type dropdown are as expected

### CODE REVIEW

File: plugin-hrm-form/src/components/tabbedForms/CSAMReportButton.tsx [hunk 1]
```diff
@@ -33,30 +33,39 @@ const CSAMReportButton: React.FC<Props> = ({
...
-        )}
-      </Row>
-      <CSAMReportDropdown
-        dropdown={dropdown}
-        handleChildCSAMType={handleChildCSAMType}
-        handleCounsellorCSAMType={handleCounsellorCSAMType}
-      />
+      <div
+        onBlurCapture={event => {
+          console.log('CSAM Dropdown Blur');
```
Comment (mythilytm) [inline plugin-hrm-form/src/components/tabbedForms/CSAMReportButton.tsx:38]: Assuming this console log was meant to be deleted

**Review by mythilytm — APPROVED** [review]
Looks great!

File: plugin-hrm-form/src/components/tabbedForms/CSAMReportDropdown.tsx [hunk 2]
```diff
@@ -6,21 +6,33 @@ import { StyledCSAMReportDropdown, StyledCSAMReportDropdownList, StyledCSAMRepor
 type OwnProps = {
   handleChildCSAMType: () => void;
   handleCounsellorCSAMType: () => void;
-  dropdown: boolean;
+  dropdownState: [boolean, (state: boolean) => void];
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/tabbedForms/CSAMReportDropdown.tsx:9]:
```suggestion
  dropdownState: ReturnType<typeof React.useState<boolean>>;
```
might be more explicit?

File: plugin-hrm-form/src/components/tabbedForms/CSAMReportDropdown.tsx [hunk 3]
```diff
@@ -6,21 +6,33 @@ import { StyledCSAMReportDropdown, StyledCSAMReportDropdownList, StyledCSAMRepor
...
 };
 
 type Props = OwnProps;
 
-const CSAMReportDropdown: React.FC<Props> = ({ handleChildCSAMType, handleCounsellorCSAMType, dropdown }) => {
+const CSAMReportDropdown: React.FC<Props> = ({
+  handleChildCSAMType,
+  handleCounsellorCSAMType,
+  dropdownState: [dropdown],
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/tabbedForms/CSAMReportDropdown.tsx:17]: Is the setter ever really used in this component?

**Review by GPaoloni — APPROVED** [review]
LGTM!

**Pull request merged**